### ISSUE TITLE
Document G93 file format

### ISSUE THREAD

**Issue by gwlucastrig** [gwlucastrig/gridfour#2]
Now that the G93 file format is relatively stable, it would be helpful to potential developers if it were documented.    Therefore, I propose that the Gridfour project create an file-format specification or some kind of Interface Control Document (ICD).

A good format specification would be particularly useful to developers translating G93 into a language other than Java.

The layout of the file could, of course, be deduced from the code. But good documentation should also cover some of the rationale for the design choices for the format as well as some of the business rules for accessing it.

For example, each file is assigned an arbitrary UUID.  This value is shared with the associated index file so that it is always possible to confirm that a particular index is the correct one for the main G93 file.   It would save developers who are studying the file format a lot of time if this idea was documented somewhere.

**Comment by gwlucastrig** [comment]
I plan to return to this Issue once Issue #10 is complete.

**Comment by gwlucastrig** [comment]
Issue is overcome by events.  A new issue will be filed for GVRS.
